src/Comments.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from "react";
+import { getCommentsByID } from "./UTILS/utils";
+import "./CSS/comments.css";
+
+export default function Comments({ articleID }) {
+  const [articleComments, setArticleComments] = useState([]);
+
+  useEffect(() => {
+    if (articleID) {
+      getCommentsByID(articleID)
+        .then((response) => {
+          console.log(response.comments);
+          setArticleComments(response.comments);
+        })
+        .catch((error) => {
+          console.error("Error fetching comments:", error);
+        });
+    }
+  }, [articleID]);
+
+  function arrangeDate(timeString) {
+    if (timeString) {
+      const timeArray = timeString.split("T");
+      return timeArray[0];
+    }
+    return "";
```

Comment:
This is good but I think these utility functions could be extracted to their own files to keep the component tidy 👍